rest/service/dashboardTemplate.go

```diff
@@ -128,6 +128,33 @@ var (
 				Title: "Recently visited",
 			},
 		},
+		models.Openshift: models.ModuleFederationMetadata{
+			Scope:    "landing",
+			Module:   "./OpenShiftAiWidget",
+			Defaults: models.BaseWidgetDimensions.InitDimensions(models.BaseWidgetDimensions{}, 1, 3, 3, 1),
+			Config: models.WidgetConfiguration{
+				Icon:  models.OpenShiftAiIcon,
+				Title: "Red Hat OpenShift AI",
+			},
+		},
+		models.Openshift: models.ModuleFederationMetadata{
+			Scope:    "landing",
+			Module:   "./QuayWidget",
+			Defaults: models.BaseWidgetDimensions.InitDimensions(models.BaseWidgetDimensions{}, 1, 3, 3, 1),
+			Config: models.WidgetConfiguration{
+				Icon:  models.QuayIcon,
+				Title: "Quay.io",
+			},
+		},
+		models.Openshift: models.ModuleFederationMetadata{
```

Comment:
Shouldn't this be `Acs`?
```suggestion
		models.Acs: models.ModuleFederationMetadata{
```